CODE_REVIEW.md

```diff
@@ -0,0 +1,33 @@
+# Code Review  
+## Code Smells  
+- action naming should be consistent - three types of actions (commands, documents, events) should each follow naming convention - a command should start with a verb (like 'ADD_ITEM'), a document should be a noun (like 'ITEM'), an event should use a past tense verb (like 'APP_WENT_ONLINE')  
```

Comment:
What about the action naming _isn't_ consistent? Can you give examples?

Reply:
using past tense verb or noun inconsistently in below examples:
`failedAddToReadingList` vs `searchBooksFailure` vs `loadReadingListError`
`confirmedAddToReadingList` vs `loadReadingListSuccess` vs `searchBooksSuccess`